14/index.php

```diff
@@ -0,0 +1,203 @@
+  <li>Made it possible to specify app compatibility modes as a user default (#3949)</li>
+  <li>Improved input reliability with modifier keys and cursor keys (#2588, #3946)</li>
+</ul>
+
+
+
+<h2 id='linux' class='red underline'>Keyman 14 for Linux</h2>
+
+<h3>What's New in Keyman 14 for Linux?</h3>
+
+<ul>
+  <li>Open a .kmp file with Keyman Config (#3183)</li>
+  <li>Now supports Ubuntu 20.10 (Groovy) (#3876)</li>
+  <li>Improved user interface</li>
+  <li>Improved support for KDE, Gnome, Fedora, Arch Linux</li>
+</ul>
+
+<li>Ubuntu, Wasta-Linux: Keyman for Linux can be installed via launchpad:</li>
+<blockquote><pre class='language-bash code'><code>sudo add-apt-repository ppa:keymanapp/keyman-nightly
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install keyman onboard</code></pre></blockquote>
+
+
+<?php
+downloadSection('Keyman for Android 14', 'android', 'keyman-$version.apk', 'beta');
+?>
+
+<p>You can also <a href=<?= $playstore_signup_link ?>>sign up</a> to access pre-release versions through Google Play.</p>
+<?= $playstoreTable ?>
+
+<h3>What's new in Keyman for Android 14?</h3>
+
+<ul>
+  <li>Improved UI for installing keyboard packages (#3498)</li>
+  <li>
+    Select a language during keyboard package installation (#3481)<br>
+    <img alt='Select language during keyboard installation' src='select_language.png'>
+  </li>
+  <li>Added new menu to add languages for an installed keyboard package (#3255)</li>
+  <li>Consolidated install menus for installing keyboards (#3245)</li>
+  <li>Fix slow input in the embedded browser (#3768)</li>
+  <li>Add system globe action to show system keyboards (#3197)</li>
+  <li>Improved correction suggestions (#3555)</li>
+  <li>Match user input capital letters when offering suggestions (#3845)</li>
+  <li> Update minimum Android SDK to 21 (Android 5.0 Lollipop) (#2993)</li>
+</ul>
+
+<h2 class="red underline">Keyman for iPhone and iPad 14</h2>
+<h3>Beta</h3>
+<p>Please register for the pre-release version through the links below for Keyman <a href=<?= $testflight_beta_link ?>>beta</a>
+  pre-releases on your iOS device. This will grant access to the app through <a  href="https://developer.apple.com/testflight/testers/">Apple's
+    Testflight app</a>, which facilitates direct installation on iOS devices.</p>
+
+<?= iosTestFlightTable(); ?>
+
+<h3>What's new in Keyman for iPhone and iPad 14?</h3>
+
+<ul>
+  <li>Select a language during keyboard package installation</li>
+  <li>Improved batching of keyboard and dictionary downloads</li>
+  <li>Improved correction suggestions (#3555)</li>
+  <li>Match user input capital letters when offering suggestions (#3845)</li>
+</ul>
+
+
+
+
+
+<?php
+downloadSection('KeymanWeb 14', 'web', 'keymanweb-$version.zip', 'beta');
+?>
+
+<h3>What's New in KeymanWeb 14?</h3>
+
+<ul>
+  <li>Add special keys for right-to-left keyboards</li>
+  <li>Block key previews for blank/hidden keys</li>
+</ul>
```

Comment:
```suggestion
  <li>Sourcemap improvements (#2809)</li>
  <li>Fixes issues with keyboard rules involving system stores (#2884)</li>
  <li>Fixes issues with keyboard rules involving both 'notany' and 'context' (#3817)</li>
</ul>
```

I'm including sourcemap stuff because the most direct 'users' of the KeymanWeb engine are website developers.  Sourcemaps provide a massive improvement over heavily minified code for debugging any integration issues.